contrib/dracut/90zfs/module-setup.sh.in

```diff
@@ -19,60 +19,39 @@ depends() {
 }
 
 installkernel() {
-	instmods zfs
-	instmods zcommon
-	instmods znvpair
-	instmods zavl
-	instmods zunicode
-	instmods zlua
-	instmods icp
-	instmods spl
-	instmods zlib_deflate
-	instmods zlib_inflate
+	instmods \
+		zfs \
+		zcommon \
+		znvpair \
+		zavl \
+		zunicode \
+		zlua \
+		icp \
+		spl \
+		zlib_deflate \
+		zlib_inflate
```

Comment:
This can probably just be `instmods zfs`

Reply:
Done!